ONSdigital/srm-rh-ui
# Updated csp policies to match ga documentation

# Motivation and Context

Wheen looking into the Ga documentation around CSP policies, we saw that we didn't have the entire list they recommend using. This PR just adds the other values to the policy that we were missing.
# What has changed



- Updated CSP policies
# How to test?



- Run locally and make sure everything in the console looks as expected.
- Run in the ssdc-rm-dev environment (Branch should already be there on srm-rh-ui) and make sure there's no error logs. 
- Check the GA console for the dev project and make sure you still see events coming
# Links



[Jira](https://jira.ons.gov.uk/browse/SDCSRM-723)
